src/tests/end-to-end/common/scan-for-accessibility-issues.ts

```diff
@@ -24,7 +24,7 @@ export async function scanForAccessibilityIssues(page: Page, selector: string):
         } as ElementContext);
     }, selector)) as AxeResults;
 
-    const violations = axeResults.violations;
+    const violations = axeResults.violations.filter(violation => violation.tags.indexOf('best-practice') === -1);
```

Comment:
You can do this as part of configuring `axe.run`'s [options parameter](https://www.deque.com/axe/axe-for-web/documentation/api-documentation/#options-parameter) to avoid having it spend the time scanning for the rules we don't want:

```typescript
axe.run(
    { include: [selectorInEvaluate] },
    { runOnly: { type: "tag", values: ["wcag2a", "wcag2aa"] } }
)
```

Reply:
Apparently you can use exclusion instead of inclusion rules with it, so maybe this is better:

```typescript
axe.run(
    { include: [selectorInEvaluate] },
    { runOnly: { type: "tag", values: {
        exclude: ["experimental", "best-practice"] } } } )
```

Reply:
I ran into some errors when I tried to exclude tags